platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -131,6 +150,178 @@ null_crypto_routine(odp_packet_t pkt ODP_UNUSED,
 	return ODP_CRYPTO_ALG_ERR_NONE;
 }
 
+static
+odp_crypto_alg_err_t aes_gcm_encrypt(odp_packet_t pkt,
```

Comment:
Neither the encryption nor the decryption function clears the hash result bytes before processing if the hash result is withing the authenticated range. ODP API specifies that the bytes are cleared before the hash calculation in that case.

Reply:
I wasn't aware this was required by the API spec. I will add the missing code for it. Thanks